src/lib/react/ReactDOMServer.hx

```diff
@@ -12,10 +14,22 @@ extern class ReactDOMServer
 	/**
 		https://facebook.github.io/react/docs/react-dom-server.html#rendertostring
 	**/
-	public static function renderToString(component:ReactComponent):String;
+	public static function renderToString(component:ReactElement):String;
 
 	/**
 		https://facebook.github.io/react/docs/react-dom-server.html#rendertostaticmarkup
 	**/
-	public static function renderToStaticMarkup(component:ReactComponent):String;
+	public static function renderToStaticMarkup(component:ReactElement):String;
+
+	#if nodejs
+	/**
+		https://reactjs.org/docs/react-dom-server.html#rendertonodestream
+	**/
+	public static function renderToNodeStream(component:ReactElement):js.node.stream.Readable<Dynamic>;
```

Comment:
Why `Dynamic`?

Reply:
Didn't know there is ReactMarkupReadableStream. Thanks!